Common/Transforms/itkAdvancedImageMomentsCalculator.h

```diff
@@ -285,7 +287,7 @@ class AdvancedImageMomentsCalculator : public Object
 
   mutable MultiThreaderParameterType m_ThreaderParameters;
 
-  mutable AlignedComputePerThreadStruct * m_ComputePerThreadVariables;
+  mutable std::unique_ptr<AlignedComputePerThreadStruct[]> m_ComputePerThreadVariables;
```

Comment:
Instead of `std::unique_ptr<AlignedComputePerThreadStruct[]>`, maybe use `std::vector<AlignedComputePerThreadStruct>`